### ISSUE TITLE
`fisher --list` with no installed plugins exits with an error

### ISSUE THREAD

**Issue by kballard** [fisherman/fisherman#38]
If nothing has been installed, `fisher --list` exits with an error and no output. It should instead tell me that there's nothing installed on stderr and exit cleanly. Maybe exiting with code 1 is reasonable, but IIRC it exited with code 127 which is weird (I can't verify since I've already installed something now).

**Comment by bucaran** [comment]
Should be easy to fix. I will also add a test for it.

**Comment by bucaran** [comment]
@kballard I am unable to verify it returns 127.

<img width="605" alt="screen shot 2016-01-11 at 5 53 24 pm" src="https://cloud.githubusercontent.com/assets/8317250/12229552/43c57afa-b88c-11e5-9a20-b7688e55909e.png">

**Comment by kballard** [comment]
Looks like it was `124`, not `127`. I can reproduce with a fresh install of fisherman on another computer:

```
~/D/fisherman (master=)> make
Adding configuration to ~/.config/fish/config.fish
Creating ~/.config/fisherman/cache
Downloading the index for the first time...
Reset your shell and type 'fisher <command>' to use Fisherman
Available Commands:
  help       Show Help 
  install    Install Plugins 
  search     Search Plugins 
  uninstall  Uninstall Plugins 
  update     Update Plugins and Fisherman 

Other Documentation:
  plugins   Creating Fisherman Plugins
  fishfile  Fishfile Format
  fisher    An Introduction to Fisherman
  faq       Frequently Asked Questions
  config    Fisherman Configuration
  commands  Creating Fisherman Commands

Use 'fisher help -g' to list guides and other documentation.
See 'fisher help <command or concept>' to access a man page.
~/D/fisherman (master=)> exec fish -l # reset the shell
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
~/D/fisherman (master=)> fisher --list
~/D/fisherman (master=) [124]> 
```

(the `[124]` in my prompt is the exit status of the last command)

**Comment by bucaran** [comment]
Thanks. This has been fixed in 0.4.0, which I am about to push.

**Comment by bucaran** [comment]
@kballard  

124 is the exit code that is set by fish when wildcard expansion fails to expand, so it makes sense you were getting that number. 

I have changed the behavior of `fisher --list` to simply return 1.
